app/Resources/views/groupManagment/detailGroup.html.twig

```diff
@@ -1,73 +1,67 @@
 {% extends "includes/base.html.twig" %}
 
-{% block title %}group{% endblock %}
+{% block title %}group !!!static, change!!!{% endblock %}
 
 {% block stylesheets %}
     {{ parent() }}
 {% endblock %}
 
-{% block body %}
-    {% include 'includes/navbar.html.twig' %}
-    <div class="container">
-        <br><br>
-        {% include 'includes/showFlashes.html.twig' %}
-        <h2>{% trans %}general.group{% endtrans %}</h2>
-        <div class="col-sm-8">
-            <table class="table table-striped">
-                <thead class="">
-                <tr>
-                    <th>
-                        {% trans %}general.name{% endtrans %}
-                    </th>
-                    <th>
-                        {% trans %}general.value{% endtrans %}
-                    </th>
-                    <th>
-                        {% trans %}general.delete{% endtrans %}
-                    </th>
-                </tr>
-                </thead>
-                <tbody>
-                <tr>
-                    <th>
-                        {% trans %}general.name{% endtrans %}
-                    </th>
-                    <td>
-                        {{group.cn}}
-                    </td>
-                    <td>
-                        -
-                    </td>
-                </tr>
+{% block content %}
+    {% include 'includes/showFlashes.html.twig' %}
+    <h2>{% trans %}general.group{% endtrans %}</h2>
+    <div class="col-sm-8">
+        <table class="table table-striped">
+            <thead class="">
+            <tr>
                 <th>
-                    {% trans %}general.gid{% endtrans %}
```

Comment:
Warum soll die gid raus?

Reply:
Die ist bei mir gar nicht raus?! Hat sich nur die Einrückung geändert und in der nächsten Zeile steht sie wieder?
Wenn du aber mal auf https://github.com/pbnl/ScoTooBo/blob/master/app/Resources/views/groupManagment/detailGroup.html.twig siehst, fällt mir ein fehlendes <tr> vor der gid auf; dieses habe ich nun ergänzt.